### DISCUSSION TITLE
Distorted Component when imported in `React`

### DISCUSSION THREAD

**Discussion by Abhishek-Mallick** [tailwindlabs/tailwindcss#12568] [Help] [answered]
Hello everyone, I'm new to Tailwind and I'm facing a issue while importing a component to my react app
The component I'm trying to import is this one [E-Commerce(first one)](https://tailwindui.com/components/ecommerce/components/store-navigation)

I have pasted it in a `Navigation.jsx` file and have imported it in my `App.js` something like this
```jsx
import './App.css';
import Navigation from './Customer/Components/Navigation/Navigation'

function App() {
  return (
    <div className="">
      <Navigation />
      <div>Home</div>
    </div>
  );
}

export default App;
```
This is how my `tailwind.config.js` looks like
```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [
    require('@tailwindcss/aspect-ratio'),
  ],
}
```
There's nothing more stylings added as of now still I'm getting this weird distorted format.
![image](https://github.com/tailwindlabs/tailwindcss/assets/106394426/eeabab6d-0713-4f9a-83a3-db198424c121)

**Comment by wongjn** [comment, marked as answer]
It seems like the `Navigation.jsx` file is not covered by your `content` file globs. Assuming the `Navigation.jsx` file is in the `src` folder, consider adjusting your Tailwind configuration like so:

```diff
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,js}"],
+. content: ["./src/**/*.{html,js,jsx}"],
   theme: {
     extend: {},
```
